### PR TITLE
Make changelog visible to docs

### PR DESCRIPTION
**Pull request by Centrus007**
I had some troubles with `recommonmark`. Ended up giving `m2r` a try which seems to have worked alright for the most part.

A little niggle is symbol handling. E.g. the markdown syntax for the right arrow differs from rst and `m2r` isn't parsing the right arrow from rst to markdown correctly :(

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,21 +1,22 @@
-# Changelog
+# Release Notes
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+<br/>
```
Comment (SimonBiggs) [inline changelog.md]: I'm surprised this line break is needed. Okay if it is, just seems unusual.

In Markdown a line break is usually included if two blank lines are left.

**Comment by SimonBiggs** [comment]
This is looking great :) thanks @Centrus007 :)

**Comment by Centrus007** [comment]
No worries!

I actually feel a little bad that I've timed such a sudden surge in contribution with your holiday! I almost think I should stop so that I can't tempt you away from your fishing :stuck_out_tongue:

**Comment by SimonBiggs** [comment]
Don't feel bad at all. I'm enjoying it. Right now I'm stuck in the car
supervising while Esther is shopping :).

I certainly won't sit at the computer unless I want to. And once at the
beach I believe it will barely occur. :).

**Comment by SimonBiggs** [comment]
And certainly please don't stop :)

**Comment by SimonBiggs** [comment]
I'll make you admin on read the docs also, in case there are issues there.

**Comment by SimonBiggs** [comment]
Can you make a readthedocs account and let me know your username

**Comment by SimonBiggs** [comment]
This looks great!

http://pymedphys.com/en/latest/additional/changelog.html

**Comment by Centrus007** [comment]
Just made a readthedocs account; username "Centrus"

**Comment by SimonBiggs** [comment]
You've been made a maintainer of readthedocs

**Comment by Centrus007** [comment]
Cool :D

**Pull request merged**